<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../logo.png</normaloff>../../logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="8">
     <widget class="QTabWidget" name="tabs">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Genome Analysis</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0">
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QSplitter" name="splitter">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QWidget" name="layoutWidget">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Functional regions:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QCheckBox" name="CDS">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>CDS</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="CENTRO">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Centrometre</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="INTRON">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Intron</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="MOBILE">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>mobile element</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QCheckBox" name="NC_RNA">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>ncRNA</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="R_RNA">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>rRNA</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="TELOMETRE">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>telometre</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="T_RNA">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>tRNA</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QCheckBox" name="UTR_3">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>3'UTR</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="UTR_5">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>5'UTR</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="ALL">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>All</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QCheckBox" name="NONE">
                       <property name="minimumSize">
                        <size>
                         <width>121</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>242</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>None</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span
                                          style=&quot; font-size:14pt;&quot;&gt;Database&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTreeView" name="treeView"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="layoutWidget">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <layout class="QFormLayout" name="formLayout_5">
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_9">
                     <property name="text">
                      <string>Scanned files</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QProgressBar" name="progressBar">
                     <property name="value">
                      <number>0</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_4">
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>Scanned organisms</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QPushButton" name="pushButton">
                     <property name="text">
                      <string>Start</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QProgressBar" name="progressBar_2">
                     <property name="value">
                      <number>0</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_6">
                   <item row="0" column="1">
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>
                                              &lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span
                                              style=&quot;
                                              font-size:14pt;&quot;&gt;Log&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="author">
       <attribute name="title">
        <string>Authors</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>7</x>
          <y>10</y>
          <width>1091</width>
          <height>531</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <item row="1" column="1">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Python&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Version</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="name">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;  ALLEMAND Fabien&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;  BONNAIL Julie&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;  COUTURE Louise&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;  LEBOT Samuel&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;  VLAYKOV Math√©o&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="txt_2">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Programming 
   language:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="txt_1">
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Developed by:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;v1.0.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;2023&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Read me</string>
       </attribute>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>93</x>
          <y>-1</y>
          <width>831</width>
          <height>551</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="8">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="bg_author">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>GenBank Analyzer</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionExporter_les_r_sultats">
   <property name="text">
    <string>Export results</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
